modApiExt/alter.lua

```diff
@@ -493,8 +502,7 @@ function modApiExtHooks:overrideAllSkills()
 				v.__Id = k
 				v.__GetSkillEffect = rawget(v, "GetSkillEffect")
 				skillIndex[k] = setmetatable(
-					{ GetSkillEffect = v.__GetSkillEffect },
-					{ __index = v }
+					{ __index = skillIndexingFn }
```

Comment:
`setmetatable` requires two arguments, so we need an empty table for the first parameter.